execution/executor/withdraw.go

```diff
@@ -29,8 +29,9 @@ func (e *WithdrawExecutor) Execute(trx *tx.Tx, sb sandbox.Sandbox) error {
 		return errors.Errorf(errors.ErrInvalidSequence,
 			"expected: %v, got: %v", val.Sequence()+1, trx.Sequence())
 	}
-	if val.Stake() < pld.Amount+trx.Fee() {
-		return errors.Error(errors.ErrInsufficientFunds)
+	if val.Stake() != pld.Amount+trx.Fee() {
```

Comment:
why it should be `!=` and not `<` ?

Reply:
check the issue

Reply:
@kehiy 
thanks, please fix the pipeline errors.